src/m365/spo/commands/site/site-commsite-enable.ts

```diff
@@ -64,10 +79,14 @@ class SpoSiteCommSiteEnableCommand extends SpoCommand {
   }
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
-    const designPackageId: string = args.options.designPackageId || '96c933ac-3698-44c7-9f4a-5fd17d71af9e';
+    const designPackageId = args.options.designPackageId || {
```

Comment:
I find this code slightly hard to read. Shall we move this to a `this.getDesignPackageId(args.options)` function? That function can return the `options.designPackageId` immediately and otherwise run a switch statement on the `options.designPackage`.